wgengine/magicsock/magicsock.go

```diff
@@ -69,6 +69,14 @@ const (
 	// _linux variant.
 	discoMagic1 = 0x5453f09f
 	discoMagic2 = 0x92ac
+
+	// UDP socket read/write buffer size (12MB). This will be set using
+	// SO_RECVBUF/SO_SNDBUF, and is often clamped by the OS to a much smaller
+	// value. This buffer includes the cost of packet headers and OS internal
+	// management structures (i.e. skb's on Linux). Linux has a special
+	// implementation to use SO_RCVBUFFORCE/SO_SNDBUFFORCE to bypass the clamp -
```

Comment:
No need to speak to implementation details here IMO, the comment will bitrot sooner if we ever change implementations.

Reply:
Ok, though it's unfortunate to not leave breadcumbs for readers. With the methods being spread about, it's almost more messy to spread it around the methods instead.

Reply:
Not a strong opinion on my part if you'd rather keep it. My usual flow is to look for uses of the constant, and from there things usually come together easily. But I don't mind that much.